catwalk/tasks/__init__.py

```diff
@@ -21,6 +21,51 @@
             correct_answer_index_field="label"
         )
     ).add_metrics(MC_METRICS),
+    "squad": HFDatasetsTask("squad").add_instance_conversion(
+        InstanceFormat.HF_QA,
+        hfqa_conversion(
+            context_field="context",
+            question_field="question",
+            answers_field="answers",
+            id_field="id"
+        )
+    ).add_metrics(QA_METRICS),
+    "squadshifts-reddit": HFDatasetsTask("squadshifts", "reddit").add_instance_conversion(
+        InstanceFormat.HF_QA,
+        hfqa_conversion(
+            context_field="context",
+            question_field="question",
+            answers_field="answers",
+            id_field="id"
+        )   
+    ).add_metrics(QA_METRICS),
+    "squadshifts-amazon": HFDatasetsTask("squadshifts", "amazon").add_instance_conversion(
+        InstanceFormat.HF_QA,
+        hfqa_conversion(
+            context_field="context",
+            question_field="question",
+            answers_field="answers",
+            id_field="id"
+        )   
+    ).add_metrics(QA_METRICS),
+    "squadshifts-nyt": HFDatasetsTask("squadshifts", "nyt").add_instance_conversion(
+        InstanceFormat.HF_QA,
+        hfqa_conversion(
+            context_field="context",
+            question_field="question",
+            answers_field="answers",
+            id_field="id"
+        )   
+    ).add_metrics(QA_METRICS),
+    "squadshifts-new-wiki": HFDatasetsTask("squadshifts", "new_wiki").add_instance_conversion(
+        InstanceFormat.HF_QA,
+        hfqa_conversion(
+            context_field="context",
+            question_field="question",
+            answers_field="answers",
+            id_field="id"
+        )   
+    ).add_metrics(QA_METRICS),
```

Comment:
I thought there'd be more QA datasets. Is this everything you ran on?

Reply:
I only added squad and squadshifts for this pr. There are more datasets once we add subsets from MRQA.